packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -13,10 +13,13 @@
                 label-style="inline"
                 is-grouped
                 :has-error="isAddressLine1Empty"
+                aria-describedby="line1-error"
```

Comment:
I'm not an expert in accessibility, unfortunately, but is this correct? If my understanding is correct, we're basically saying that the error message is what describes the form field, and that doesn't sounds correct to me.
cc @ross-gratton

Reply:
It sounds reasonable to me, assuming the error message only exists on the DOM when there is an error.

Reply:
from what I've read, `aria-describedby` is used for error messages quite frequently in accessibility - it's a way of connecting the input field with the corresponding id in the error message  - so that if the error message shows, it also reads out that information. Some forms also add `aria-invalid=true` to the input field so I could do that too. In most cases, from what I've seen, either `aria-describedby` or `aria-labelledby` is used though